### ISSUE TITLE
MultiMC FTB Import Walkthrough

### ISSUE THREAD

**Issue by Unoyahara** [Iskallia/Vault-public#1078]
Since so many people keep having issues with figuring out the import i made a guide that ill post here.

Step 1: make sure FTB and MultiMC are updated to the latest.
Step 2: open FTB, go to the magnifying glass and click this slider
![image](https://user-images.githubusercontent.com/112496489/191861900-5871a06a-8c90-4921-a53c-1db1a2c9a4d9.png)
Step 3: Search for Vault Hunters, this should take less than a minute or 2.
Step 4: Install the Modpack and let it completely install.
Step 5: Open MultiMC and hit Add Instance
Step 6: Go to FTB Import and it should now be in the list of importable Modpacks alongside whatever other modpacks you are using. this works for every other modpack you want as long as you follow these steps as well!

**Comment by JoseFernandoInsper** [comment]
I cant find VH in the FTB, this stil works ?

**Comment by Unoyahara** [comment]
Yes, it should work still, If it doesn't please let me know and I'll find
the fix and update this accordingly.

Unoyahara

**Comment by JoseFernandoInsper** [comment]
my broblem is that i cant find the VH on the FTB, only apears some modpacks but not the VH one

**Comment by JoseFernandoInsper** [comment]
i changed to the curseForge modpacks
![image](https://user-images.githubusercontent.com/18488204/194731895-822384d0-4f7f-41e4-af4f-05959b366b42.png)

**Comment by Unoyahara** [comment]
Did you switch the setting over from FTB to CurseForge?

If it's just showing modpacks like Omnia or other FTB modpacks then it
hasn't been switched over from the FTB search.

Unoyahara

**Comment by Unoyahara** [comment]
You need to actively search for Vault Hunters, it's only showing the TOAT
modpacks unless you search for it.

Unoyahara

**Comment by JoseFernandoInsper** [comment]
ah ok, i was trying to search part of the name like vault, so it was showing nothing, thank you

**Comment by Unoyahara** [comment]
No worries! Glad to have been somewhat of a help.

Unoyahara
